### PR TITLE
Rename to Editor.js

### PR DESCRIPTION
**Pull request by gohabereg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,20 +149,20 @@ Choose the most usable method of getting Editor for you.
 Install the package via NPM or Yarn
 
 ```shell
-npm i codex.editor --save-dev
+npm i editorjs --save-dev
```
Comment (neSpecc) [inline README.md]: не так

File: README.md [hunk 2]
```diff
@@ -149,20 +149,20 @@ Choose the most usable method of getting Editor for you.
...
 
 ```javascript
-const CodexEditor = require('codex.editor');
+const EditorJS = require('editorjs');
 ```
 
 ### Use from CDN
 
-You can load specific version of package from [jsDelivr CDN](https://www.jsdelivr.com/package/npm/codex.editor).
+You can load specific version of package from [jsDelivr CDN](https://www.jsdelivr.com/package/npm/editorjs).
```
Comment (neSpecc) [inline README.md]: тут тоже нужна организация?

File: src/components/modules/saver.ts [hunk 3]
```diff
@@ -68,7 +67,7 @@ export default class Saver extends Module {
     let totalTime = 0;
     const blocks = [];
 
-    console.groupCollapsed('[CodexEditor saving]:');
+    console.groupCollapsed('[EditorJS saving]:');
```
Comment (neSpecc) [inline src/components/modules/saver.ts]:
```suggestion
    console.groupCollapsed('[Editor.js saving]:');
```

File: webpack.config.js [hunk 4]
```diff
@@ -57,7 +57,7 @@ module.exports = (env, argv) => {
       }),
 
       new webpack.BannerPlugin({
-        banner: `Codex Editor\n\n@version ${VERSION}\n\n@licence Apache-2.0\n@author CodeX-Team <https://ifmo.su>\n\n@uses html-janitor\n@licence Apache-2.0 (https://github.com/guardian/html-janitor/blob/master/LICENSE)`
+        banner: `Editor.js\n\n@version ${VERSION}\n\n@licence Apache-2.0\n@author CodeX-Team <https://ifmo.su>\n\n@uses html-janitor\n@licence Apache-2.0 (https://github.com/guardian/html-janitor/blob/master/LICENSE)`
```
Comment (talyguryn) [inline webpack.config.js]:
```suggestion
        banner: `Editor.js\n\n@version ${VERSION}\n\n@licence Apache-2.0\n@author CodeX <https://codex.so>\n\n@uses html-janitor\n@licence Apache-2.0 (https://github.com/guardian/html-janitor/blob/master/LICENSE)`
```

File: package.json [hunk 5]
```diff
@@ -1,13 +1,20 @@
 {
-  "name": "codex.editor",
-  "version": "2.9.5",
-  "description": "CodeX Editor. Native JS, based on API and Open Source",
-  "main": "dist/codex-editor.js",
+  "name": "@editorjs/editorjs",
+  "version": "2.10.0",
+  "description": "Editor.js - Native JS, based on API and Open Source",
```
Comment (talyguryn) [inline package.json]:
```suggestion
  "description": "Editor.js — Native JS, based on API and Open Source",
```

**Comment by Adonai** [comment]
How do I search for this editor in Google/DDG now? New name is totally non-specific :D

**Pull request merged**